### PR TITLE
Build and upload miniforge3 image

### PR DESCRIPTION
**Pull request by xhochy**
cc @yuvipanda

### CODE REVIEW

**Comment by yuvipanda** [comment]
Awesome!

We should tag these by conda version. Does this already do it?

**Comment by xhochy** [comment]
No, this only pushes the `latest` tag yet. Tagging for the conda version will need more magic in the Travis build.

**Comment by yuvipanda** [comment]
Awesome, thanks!

File: .circleci/run_docker_build.sh [hunk 1]
```diff
@@ -17,18 +17,22 @@ if hash docker-machine 2> /dev/null && docker-machine active > /dev/null; then
     export HOST_USER_ID=$(docker-machine ssh $(docker-machine active) id -u)
 fi
 
-ARTIFACTS="$FEEDSTOCK_ROOT/build_artifacts"
+if [[ "${TEST_CONTAINER:"1"}" != "0" ]]; then
```
Comment (xhochy) [inline .circleci/run_docker_build.sh:20]: Locally this works for me but in CI I get an error about an unbound variable. Does someone know how to do that correctly?

**Comment by xhochy** [comment]
@conda-forge/core This is ready for review/merge.

**Comment by scopatz** [comment]
No need to wait for @isuruf - this looks right to me. Thanks!

**Comment by yuvipanda** [comment]
Thanks, @scopatz! Any thoughts on how we can get versioned images up?

**Pull request merged**